lib/rentlinx/validators/phone_validator.rb

```diff
@@ -11,18 +11,12 @@ def processed_value
     private
 
     def validate
-      return if value_blank? ||
-                valid_us_number_without_country_code?(processed_value) ||
-                valid_us_number_with_country_code?(processed_value)
+      return if value_blank? || valid_phone_number?(processed_value)
       @error = "#{@value} is not a valid phone number"
     end
 
-    def valid_us_number_with_country_code?(number)
-      number.start_with?('1') && valid_us_number_without_country_code?(number[1..-1])
-    end
-
-    def valid_us_number_without_country_code?(number)
-      !number.start_with?('1') && !number.start_with?('0') && number.size == 10
+    def valid_phone_number?(number)
```

Comment:
It might be good to leave "us" in the name (i.e. `valid_us_phone_number?`) as the regex is US specific.  Although I suppose Rentlinx probably only has US partners currently so maybe it doesn't matter too much.